pkg/operator/controller/cacert/controller.go

```diff
@@ -11,52 +11,88 @@ import (
 	"math/big"
 	"time"
 
+	logf "github.com/openshift/cluster-ingress-operator/pkg/log"
+
 	corev1 "k8s.io/api/core/v1"
+
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+
+	"k8s.io/client-go/tools/record"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+var log = logf.Logger.WithName("cacert-ontroller")
+
 const (
 	// caCertSecretName is the name of the secret that holds the CA certificate
 	// that the operator will use to create default certificates for
 	// clusteringresses.
 	caCertSecretName = "router-ca"
 )
 
-// routerCASecretName returns the namespaced name for the router CA secret.
-func routerCASecretName(namespace string) types.NamespacedName {
+// CASecretName returns the namespaced name for the router CA secret.
+func CASecretName(operatorNamespace string) types.NamespacedName {
```

Comment:
TODO: This seems like a possible recipe for cyclic dependencies, haven't thought much about where the right place to put naming functions might be.